ocs_ci/ocs/constants.py

```diff
@@ -501,8 +501,13 @@
 # Upgrade related constants, keeping some space between, so we can add
 # additional order.
 ORDER_BEFORE_UPGRADE = 10
-ORDER_UPGRADE = 20
-ORDER_AFTER_UPGRADE = 30
+ORDER_BEFORE_OCS_UPGRADE = 20
+ORDER_BEFORE_OCP_UPGRADE = 30
+ORDER_OCP_UPGRADE = 40
+ORDER_OCS_UPGRADE = 50
+ORDER_AFTER_OCP_UPGRADE = 60
+ORDER_AFTER_OCS_UPGRADE = 70
+ORDER_AFTER_UPGRADE = 70
```

Comment:
When I am looking at this I think that more better ordering will be 
```python
ORDER_BEFORE_UPGRADE = 10
ORDER_BEFORE_OCP_UPGRADE = 20
ORDER_OCP_UPGRADE = 30
ORDER_AFTER_OCP_UPGRADE = 40
ORDER_BEFORE_OCS_UPGRADE = 50
ORDER_OCS_UPGRADE = 60
ORDER_AFTER_OCS_UPGRADE = 70
ORDER_AFTER_UPGRADE = 8
```

WDYT @ebenahar @fbalak @AviadP

Reply:
How about having the order configurable in ENV conf with default set to OCP first?

Reply:
lgtm

Reply:
this are defined constants and this change is not going to be part of this PR.  This we discussed some time back but will be part of different PR and we need to figure out the approach which will work here. Anyway those are constants and can be used as default values if not provided by config. Then I can make it like read config values first if provided if not we will use those values.

Reply:
Ok